packages/transactions/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -80,19 +80,74 @@ export const OutboundShipmentListView: FC = () => {
         mutateAsync({ ...row, color: color.hex });
       }),
 
-      key: 'customer',
-      label: 'label.customer',
+      key: 'name',
+      label: 'label.name',
       sortable: false,
+      width: 150,
+      minWidth: 150,
+      maxWidth: 250,
+      align: 'left',
     },
     {
-      label: 'label.id',
-      key: 'id',
-      sortable: false,
+      label: 'label.type',
+      key: 'type',
+      width: 100,
+      minWidth: 100,
+      maxWidth: 100,
+      align: 'left',
+    },
+    {
+      label: 'label.status',
+      key: 'status',
+      width: 75,
+      minWidth: 75,
+      maxWidth: 75,
+      align: 'left',
+    },
+    {
+      label: 'label.entered',
+      key: 'entered',
+      format: ColumnFormat.date,
+      width: 75,
+      minWidth: 75,
+      maxWidth: 75,
+      align: 'left',
+    },
+    {
+      label: 'label.confirmed',
+      key: 'confirmed',
+      format: ColumnFormat.date,
+      width: 75,
+      minWidth: 75,
+      maxWidth: 75,
+      align: 'left',
     },
-    { label: 'label.date', key: 'date', format: ColumnFormat.date },
-    { label: 'label.customer', key: 'customer' },
-    { label: 'label.supplier', key: 'supplier' },
-    { label: 'label.total', key: 'total' },
+
+    {
+      label: 'label.invoice-number',
+      key: 'invoiceNumber',
+      width: 25,
+      minWidth: 25,
+      maxWidth: 25,
+      align: 'left',
+    },
+    {
+      label: 'label.total',
+      key: 'total',
+      width: 75,
+      minWidth: 75,
+      maxWidth: 75,
+      align: 'right',
+    },
+    {
+      label: 'label.comment',
+      key: 'comment',
+      width: 150,
+      minWidth: 300,
+      maxWidth: 450,
+      align: 'left',
+    },
```

Comment:
I couldn't decide what to do with these.

Could put all in a `columns` folder in the `tables` folder and use `getCommentColumn` and pass params.. put them all into a file in the `transactions` package or keep them in this file, leave them here 🤔

Reply:
me too! 

leaning towards putting them in a shared file and using those column definitions to compose the array of columns here. or possibly going full mobile and having collections of columns already defined - though there has been a few issues with the inflexibility of that ( such as combinations of preferences..using cash handling and vaccines and programs )

I think there's value in having each column look the same, regardless of which specific table it is used in. such as having the 'confirmed date' always with a particular format and width. 🤷‍♂️